app/src/main/java/org/wikiedufoundation/wikiedudashboard/di/repositoryModule.kt

```diff
@@ -0,0 +1,16 @@
+package org.wikiedufoundation.wikiedudashboard.di
+
+import org.koin.dsl.module
+import org.wikiedufoundation.wikiedudashboard.ui.campaign.Repository.ActiveCampaignRepository
+import org.wikiedufoundation.wikiedudashboard.ui.campaign.dao.ActiveCampaignDao
+import org.wikiedufoundation.wikiedudashboard.data.network.WikiEduDashboardApi
+
+val repositoryModule = module {
+
+    fun provideCampaignListRepository(api:WikiEduDashboardApi, activeCampaignDao: ActiveCampaignDao)
+            : ActiveCampaignRepository{
+        return ActiveCampaignRepository(api, activeCampaignDao)
```

Comment:
Since it's one line only with return you can remove the `{ ... }` and use `=`
fun provide...() = ActiveCampaignRepository(api, activeCampaignDao)